macros/src/main/scala/me/rexim/morganey/meta/package.scala

```diff
@@ -0,0 +1,19 @@
+package me.rexim.morganey
+
+import scala.language.experimental.macros
+
+package object meta extends AnyRef
+  with DefaultLiftableInstances
+  with DefaultUnliftableInstances {
+
+  private[meta] trait Quasiquotation {
+    def apply[T](args: T*): Any = macro QuotationMacro.quote
+    def unapply(arg: Any): Any = macro QuotationMacro.unquote
+  }
+
+  implicit class Quasiquote(ctx: StringContext) {
+    object m extends Quasiquotation
+    object lc extends Quasiquotation
```

Comment:
I think `m` would be a better name, because it's a reference of the projects name. It would be possible to rename it, to some longer name: `mor` or `mg`, or something like that. If you really wanted it could also be renamed to `morganey`, but I think that'll be too long for the name for an interpolator.

I'm not quite sure why you think single-letter interpolators is standard library's prerogative, after all Scala let's you give them arbitrary names .